### ISSUE TITLE
Add one-off links to feed

### ISSUE THREAD

**Issue by mwainwright** [nwspk/lcpt_twitter_bot#3]
A nice feature would be the ability to inject a link once only into the regular Twitter stream without retaining it in the Library. In particular this would be useful for links to forthcoming events, resources only temporarily available, job postings, etc.

**Comment by edsaperia** [comment]
If you control the twitter account, you can always just tweet from it manually. Or do you mean something different?

**Comment by mwainwright** [comment]
True. But I don't! Also I was imagining something that would go in the regular rhythm of tweets, and perhaps that could be repeated up to some end date; e.g. a conference next year could be included with a date (in a tag) up until which it should be tweeted with other links in the library, and after which deleted.
